jsapp/js/rootContextProvider.component.tsx

```diff
@@ -0,0 +1,11 @@
+import React, {ReactNode} from 'react';
+import {BillingContextProvider} from 'js/account/billingContextProvider.component';
+
+/* This is a context provider that wraps the root element (in ./app.js)
+ * Use this if you need to share context between, for example, the sidebar and the main page
```

Comment:
"the sidebar and the main page" is a little vague and if we add a signed out state, this won't be true. Maybe just emphasize using this as little as possible and only when it's truly needed as global state. 

Imagine I make a new Foo Feature with a foo page and a foo nav. I may need a foo context. But I don't need to add it to RootContext.